README.md

```diff
@@ -4,7 +4,7 @@ To start your Phoenix server:
 
   * Install dependencies with `mix deps.get`
   * Install Node.js dependencies with `cd assets && npm install`
-  * Start Phoenix endpoint with `mix phx.server`
+  * Start Phoenix endpoint with `USERNAME=username PASSWORD=password mix phx.server` (substituting whatever username/password you want)
```

Comment:
Yeah, I don't really see the harm in this. Alternatively, we could just mention that `USERNAME` and `PASSWORD` are environment variables. Also maybe mention that it's run via `mix run --no-halt`